backends/platform/sdl/sdl-window.cpp

```diff
@@ -162,6 +162,21 @@ void SdlWindow::toggleMouseGrab() {
 #endif
 }
 
+bool SdlWindow::lockMouse(bool lock) {
+#if SDL_VERSION_ATLEAST(2, 0, 0)
+	if (lock)
+		SDL_SetRelativeMouseMode(SDL_TRUE);
+	else
+		SDL_SetRelativeMouseMode(SDL_FALSE);
+#else
+	if (lock)
```

Comment:
with this part it will break grab/capture mouse. I tested this with SDL1. As it's same API for capture/grab mouse.